static/offline.html

```
@@ -1,6 +1,6 @@
 {{/* Offline fallback page — shown when SW can't reach network. */}}
 <!DOCTYPE html>
-<html lang="{{ site.LanguageCode | default "ru" }}" data-theme="dark">
+<html lang="{{ site.Language.Locale | default "ru" }}" data-theme="dark">
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)